src/executor.rs

```diff
@@ -655,9 +655,11 @@ fn parse_result(
         | CoreTypeConcrete::Circuit(_)
         | CoreTypeConcrete::RangeCheck96(_) => native_panic!("not yet implemented as results"),
```

Comment:
I would add change this error message too:

```suggestion
        | CoreTypeConcrete::RangeCheck96(_) => native_panic!("range check 96 not yet implemented as results"),
```